docs/index.rst

```diff
@@ -0,0 +1,24 @@
+.. MIS Builder documentation master file, created by
+   sphinx-quickstart on Sat Apr 14 14:08:55 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to MIS Builder's documentation!
+=======================================
+
+Management Information System reports for Odoo: easily build super fast,
+beautiful, custom reports such as P&L, Balance Sheets and more.
+
+.. toctree::
+   :maxdepth: 3
+   :caption: Contents:
+
```

Comment:
I suggest to add the following (to be tested though in readthedoc):
```
.. sectnum::

.. header:: |DOCNO| - |TITLE| - |DATE|

.. footer:: Copyright Creative Common - Odoo Community Association
```

Reply:
These directives must not be used with sphinx. 
* section numbering is done by adding `:numbered:` in the toctree directive
* header/footer are generated from `conf.py` variables and the theme/templates